.ci/travis/travis.sh

```diff
@@ -443,6 +443,21 @@ no-exception-test-alot-of-project1)
       --config checks-nonjavadoc-error.xml --checkstyleVersion $CS_POM_VERSION
   ;;
 
+
+no-error-pmd)
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  echo CS_version: ${CS_POM_VERSION}
+  mkdir -p .ci-temp/
+  cd .ci-temp/
+  git clone https://github.com/pmd/pmd.git
+  cd pmd
+  mvn -e install checkstyle:check -Dcheckstyle.version=${CS_POM_VERSION}
+  find -delete
+  cd ../
+  rmdir pmd
```

Comment:
please make a function for removal of folder with proptected files.

Reply:
@romani how would this be?
```removeFolderWithProtectedFiles() {
  while [[ "$PWD" != '/' && ${PWD##*/} != "$1" ]];
  do
    cd ..
    echo "${PWD##*/}"
  done
  if [[ "${PWD##*/}" == "$1" ]]; then
    find . -delete
    cd ..
    rmdir "$1"
    echo "$1 folder removed..."
    return 0;
  fi
  echo "unable to remove $1 folder"
}
```

Reply:
I thought to make it simple
```
cd $1
find . -delete
cd ..
rmdir "$1"
```
Complicated logic in shell is very fragile and hard to support.

Reply:
I agree; however, some of the no-error-test projects are a few subdirectories deep like https://github.com/checkstyle/checkstyle/blob/341ef2d7296242b03bff99f37454784ccea82906/.ci/wercker.sh#L156 and https://github.com/checkstyle/checkstyle/blob/341ef2d7296242b03bff99f37454784ccea82906/.ci/wercker.sh#L169.  I felt the recursive check for correct directory would be necessary for those like this.

Reply:
.git folder is always at same place, it is completely different fact that we in repo use different folder for validation. 
We created repo folder we should clean up it. We should not care why we did inside repo.

Reply:
Let's try simple first and improve later on

Reply:
Fixed.